frontend/src/modules/app/ToolBar.tsx

```diff
@@ -163,74 +154,69 @@ export const ToolBar = ({
   const alarmLimitsRequestDraftSelect = useSelector(getAlarmLimitsRequestDraft);
   const alarmLimitsRequestSelect = useSelector(getAlarmLimitsRequest);
   const alarmLimitsRequestUnsaved = useSelector(getAlarmLimitsRequestUnsaved);
-  const alarmLimitsUnsavedKeys = useSelector(getAlarmLimitsUnsavedKeys);
   const alarmLimitsRequest = (alarmLimitsRequestSelect as unknown) as Record<string, Range>;
   const alarmLimitsRequestDraft = (alarmLimitsRequestDraftSelect as unknown) as Record<
     string,
     Range
   >;
   const ventilatingStatus = useSelector(getVentilatingStatusChanging);
```

Comment:
Let's rename this to `ventilatingStatusChanging` so that the name isn't misleading.